JsonLogic/Rules/LiteralRule.cs

```diff
@@ -48,6 +48,6 @@ internal class LiteralRuleJsonConverter : JsonConverter<LiteralRule>
 
 	public override void Write(Utf8JsonWriter writer, LiteralRule value, JsonSerializerOptions options)
 	{
-		JsonSerializer.Serialize(writer, value.Value, options);
+		JsonSerializer.Serialize(writer, value.Value, LogicSerializerContext.Default.JsonNode!);
```

Comment:
Same comment about wanting an options.Write overload here so we get type safety but also preserving options.